iscsi/initiator.go

```diff
@@ -266,7 +278,7 @@ func CleanupScsiNodes(target string, ne *util.NamespaceExecutor) error {
 					return fmt.Errorf("Failed to cleanup empty SCSI node file %v: %v", file, err)
 				}
 				// We're trying to clean up the upper level directory as well, but won't mind if we fail
-				ne.Execute("rmdir", []string{filepath.Dir(file)})
+				_, _ = ne.Execute("rmdir", []string{filepath.Dir(file)})
```

Comment:
If we don't care, why do we need this?

Reply:
By explicitly ignoring the error, we communicate via the code that we don't care.
And it's not just an oversight, for example goland and other checkers have checks for things like this.